### DISCUSSION TITLE
Logic Pro X to Facebook live stream

### DISCUSSION THREAD

**Discussion by yruamazepol** [ExistentialAudio/BlackHole#117] [General]
I am wondering how do I set up logic to have its output sound into facebook's livestream microphone AND be able to monitor that same output sound into my monitors so that I can monitor the music as well. I'm on High Sierra OS (MAC)

**Comment by devinroth** [comment]
I plan on creating a walkthrough for that soonish. But here is the quick run down.

Create Aggregate with speakers and BH
Set input/output in Logic to Aggregate
Route MIC and Logic audio to BH and Speakers via Aggregate.
